scripts/add_to_board.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+
+from github import Github
+
+from common import env
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--token", type=str, required=True)
+    parser.add_argument("--pull_request", type=int, required=True)
+    args = parser.parse_args()
+
+    repo = env("GITHUB_REPOSITORY")
+
+    g = Github(args.token)
+    repo = g.get_repo(repo)
+    pr = repo.get_pull(args.pull_request)
+
+    project = repo.get_projects()[0]
```

Comment:
Let's don't rely on indices here. Is it possible to use some id here (our project has `1` - https://github.com/intellij-rust/intellij-rust/projects/1) or at least name?

Reply:
I figured out that all these objects have their own long IDs, now replaced. But with `repo.get_projects()` there is no way to get the project by its name or id (except logic that the first project is a number 0 and so on with indexes)